.github/stale_example.yml

```diff
@@ -0,0 +1,69 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+ daysUntilStale: 50
```

Comment:
Let's consider 90 days delay (3 months)

Reply:
Agree. Let's keep 90 for now
We can decrease it later